pkgs/development/python-modules/zodb/default.nix

```diff
@@ -18,6 +19,7 @@
 buildPythonPackage rec {
   pname = "zodb";
   version = "6.0";
+  pyproject = true;
 
   src = fetchPypi {
     pname = "ZODB";
```

Comment:
can you use fetchFromGitHub/GitLab/whever instead? I believe it is preferred now